### PR TITLE
[URGENT] Added nic effective-nsg and effective-route-tables

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@EvgenyAgafonchikov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by MikhailTryakhov** [comment]
LGTM

File: lib/commands/arm/network/network._js [hunk 1]
```diff
@@ -1017,6 +1017,42 @@ exports.init = function (cli) {
       nic.delete(resourceGroup, name, options, _);
     });
 
+  var nicEffectiveRouteTable = nic.category('effective-route-table')
+    .description($('Commands to manage effective route table'));
+
+  nicEffectiveRouteTable.command('get [resource-group] [nic-name]')
```
Comment (amarzavery) [inline lib/commands/arm/network/network._js]: the name of the command should be show. Please use the approved verbs. It is documented in the documentation folder

Comment (EvgenyAgafonchikov) [inline lib/commands/arm/network/network._js]: updated

**Comment by amarzavery** [comment]
@EvgenyAgafonchikov - can you please resolve merge conflicts, so i can merge the PR.

**Comment by MikhailTryakhov** [comment]
@amarzavery  thank you for review! Ready to merge

**Comment by yugangw-msft** [comment]
/CC: @burtbiel, @tjprescott . Do we have this in azure-cli?

**Comment by tjprescott** [comment]
@yugangw-msft yes, we have these commands.

**Pull request merged**